docs/explanations.rst

```diff
@@ -28,6 +28,7 @@ selection of some of the commands -- what they do, and how they work.
 
     explanations/how_to_interpret_the_security_status_command.md
     explanations/how_to_interpret_output_of_unattended_upgrades.md
+    explanations/how_to_interpret_output_of_fix_execute_api.md
```

Comment:
I have some thoughts about how this. We now have three of these "interpret the output" pages, which is now enough for a category of its own. However, that approach would mean that eventually we might have an explanation page for almost every API endpoint and I'm not sure that's an efficient use of real estate.

Alternatively, I'm sort of wondering why we don't have this content in the API doc itself? The main reason being that if we duplicate the contents of the output in two places manually, there's big potential for them to drift out of sync with each other over time. I'm all in favour of making things easier for you guys to maintain (i.e., with one place to update instead of multiple).

Reply:
Another alternative is that we could consolidate the "fix" content onto a single page (fix plan, fix execute).

Reply:
I agree that at least for this explanation, all of the content of the explanation can be moved into the field descriptions within api.rst

Reply:
I agree, I will move it to the `api.rst`